crates/blockifier/oz_erc20/Scarb.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "oz_erc20"
+version = "0.1.0"
+edition = "2023_10"
+
+# See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest.html
+
+[dependencies]
+#openzeppelin = { path = "../../../../oz-cairo-contracts" }
+#openzeppelin = { path = "./cairo-contracts" }
```

Comment:
Nitpick, but these should probably be removed before we submit this

Reply:
I left it because it'd easier for us to debug and test (just uncomment instead of rewriting everything) but I will remove it